### DISCUSSION TITLE
paymentProcessing "extra" TAG hidden

### DISCUSSION THREAD

**Discussion by Reggerriee** [coinfoundry/miningcore#1110] [General]
Hello,

How can I hide paymentProcessing "extra" TAG? I don't want it to appear on the API.

"paymentProcessing": {
        "enabled": true,
        "minimumPayment": 0.25,
        "payoutScheme": "PPLNS",
        "payoutSchemeConfig": [
          [
            []
          ]
        ],
        **"extra": {
          "walletPassword": "xxxxxxx",
          "minimumPaymentToPaymentId": 1
        }**
      }

Thanks for your help! :)

**Comment by oliverw** [comment]
What coin type is this for?

**Reply by Reggerriee** [reply]
ergo :)

**Reply by oliverw** [reply]
The wallet password will be hidden from the API in the next release. [44ab4de9956243c6e53bc106335207bba4464634](https://github.com/oliverw/miningcore/commit/44ab4de9956243c6e53bc106335207bba4464634#diff-dc184cffeeb142fb679b493c1cbe94d025fc5d736fdef311f4b6d386968a35acR33)

**Reply by Reggerriee** [reply]
Okay thank you! :)
